src/components/pages/Dashboard/index.js

```diff
@@ -1,22 +1,43 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import '../../../styles/InstructorStyles/index.less';
-import { Layout } from 'antd';
-import InstructorSidebar from '../InstructorHome/InstructorSidebar';
-import InstructorStats from './InstructorStats';
+import { useOktaAuth } from '@okta/okta-react';
+import { getCurrentUser } from '../../../redux/actions/userActions';
+import AdminHome from '../AdminHome/index';
+import InstructorHome from '../InstructorHome/index';
+import ParentHome from '../ParentHome/ParentHome';
 
-const { Content } = Layout;
-const InstructorDashboard = () => {
-  return (
-    <div>
-      <Layout>
-        <InstructorSidebar />
-        <Content>
-          <InstructorStats />
-          {/* Other Dashboard Components */}
-        </Content>
-      </Layout>
-    </div>
-  );
+import { connect, useDispatch } from 'react-redux';
+
+const InstructorDashboard = props => {
+  const dispatch = useDispatch();
+  const { authState, oktaAuth } = useOktaAuth();
+  const { idToken } = authState;
+  const { role_id } = props.user.currentUser;
+
+  useEffect(() => {
+    if (!role_id) {
+      dispatch(getCurrentUser(authState.idToken.idToken, oktaAuth));
+    }
+  }, [dispatch, idToken]);
+
+  switch (role_id) {
+    case 1: // SuperAdmin
+      return <AdminHome />;
+    case 2: // Admin
+      return <AdminHome />;
+    case 3: // instructor
+      return <InstructorHome />;
+    case 4: // Parent
+      return <ParentHome />;
+    case 5: // Child
+      return <ParentHome />; // ChildHome does not exist
+    default:
+      return console.log('Role_ID not Found');
+  }
+};
+
+const mapStateToProps = state => {
+  return { user: state.userReducer };
 };
 
-export default InstructorDashboard;
+export default connect(mapStateToProps)(InstructorDashboard);
```

Comment:
This should be exporting `Dashboard.js`, no? 😄

Reply:
Alright just pushed these changes